### DISCUSSION TITLE
How can i use BModal, BPopover and some other directives

### DISCUSSION THREAD

**Discussion by hencework** [cdmoro/bootstrap-vue-3#2088] [Q&A] [answered]
How can i use BModal, BPopover and some other directives? i can't find in doc

**Comment by VividLemon** [comment, marked as answer]
First you read https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/packages/bootstrap-vue-next/src/directives/BPopover.ts 

Then create a PR to change these https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/apps/docs/src/docs/directives/BPopover.md

Then you go to https://bootstrap-vue-next.github.io/bootstrap-vue-next/docs/directives/BPopover.html and then you've successfully found the doc 🎉 🍾

**Reply by hencework** [reply]
Thanks for your response, I will definitely contribute to the doc.
